server/users.ts

```diff
@@ -1171,6 +1171,7 @@ class User extends Chat.MessageContext {
 				this.autoconfirmed = this.userid;
 			} else {
 				delete usergroups[this.userid];
+				this.trusted = '';
```

Comment:
Should this not be `null`?

Reply:
Looks like `''` is correct.

Reply:
`this.trusted = '';` is the default value that's set in the `User` constructor (all the other string properties are also initialized as '')